### PR TITLE
Fixing formatting for objects vs general terms

### PR DESCRIPTION
**Pull request by bergerhoffer**
Fixing formatting on terms for translation.

### CODE REVIEW

File: modules/nodes-nodes-audit-log-basic.adoc [hunk 1]
```diff
@@ -20,13 +20,13 @@ Audit works at the API server level, logging all requests coming to the server.
...
+|`objectRef` |Optional. The object reference this request is targeted at. This does not apply for `List`-type requests, or non-resource requests.
+|`responseStatus` |Optional. The response status, populated even when the `ResponseObject` is not a `Status` type. For successful responses, this will only include the code. For non-status type error responses, this will be auto-populated with the error message.
+|`requestObject` |Optional. The API object from the request, in JSON format. The `RequestObject` is recorded as is in the request (possibly re-encoded as JSON), prior to version conversion, defaulting, admission or merging. It is an external versioned object type, and might not be a valid object on its own. This is omitted for non-resource requests and is only logged at request level and higher.
+|`responseObject` |Optional. The API object returned in the response, in JSON format. The `ResponseObject` is recorded after conversion to the external type, and serialized as JSON. This is omitted for non-resource requests and is only logged at response level.
```
Comment (mburke5678) [inline modules/nodes-nodes-audit-log-basic.adoc:26]: Is the difference in capitalization correct here between the field column and the description for 24, 25, 26?
'requestObect` in the field column, `RequestObject` in the description
'responseObect` in the field column, `ResponseObject` in the description

**Comment by mburke5678** [comment]
@bergerhoffer One question. Otherwise LGTM

File: modules/nodes-nodes-audit-log-basic.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/nodes-nodes-audit-log-basic.adoc:26]: Yes it is. ResponseObject is the object, but the field that shows in the logs is responseObject. Thanks for checking!

**Pull request merged**